build/postgres-ha/Dockerfile

```diff
@@ -94,6 +94,9 @@ RUN chmod +x /opt/crunchy/bin/yq
 # volume permissions are applied when building the image
 VOLUME ["/pgdata", "/pgwal", "/pgconf", "/backrestrepo", "/sshd"]
 
+# Defines a unique directory name that will be utilized by the nss_wrapper in the UID script
+ENV NSS_WRAPPER_SUBDIR="postgres-ha"
+
```

Comment:
Are we unable to leverage this from the `postgres` container?

Reply:
See comment for `postgres-gis-ha`.